src/icon-button/types.ts

```diff
@@ -1,5 +1,10 @@
 export interface IconButtonProps
     extends React.ButtonHTMLAttributes<HTMLButtonElement> {
-    focusHighlight?: boolean;
-    focusOutline?: "none" | "browser";
+    "data-testid"?: string | undefined;
+    buttonStyle?: StyleType;
```

Comment:
rename the prop as per the previous comment
```suggestion
    styleType?: StyleType;
```